sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -174,13 +174,21 @@ export class EventHubReceiver extends LinkEntity {
    * @property {CheckpointData} _checkpoint Describes metadata about the last message received.
```

Comment:
This comment no longer matches the property type. Do we still need the `CheckpointData` interface with this change?

Reply:
I believe `CheckpointData` is no longer needed